### DISCUSSION TITLE
[Question] Tiled Cameras with Different Resolutions

### DISCUSSION THREAD

**Discussion by ori-gadot** [isaac-sim/IsaacLab#1073] [Q&A]
### Question

Is it possible to run an environment with two tiled cameras that have different width and height settings? At the moment, I receive a warning when attempting this, and the smaller images are being tiled incorrectly.

**Comment by garylvov** [comment]
I've only been able to do tiled cameras of the same resolution. As a messy workaround for the time being, you could use the highest resolution for all cameras and downsample one or more of the cameras to be of the smaller desired resolution, but of course, this has a performance overhead.

Maybe @kellyguo11 @pascal-roth  @Dhoeller19 have a more robust fix

**Comment by garylvov** [comment]
I can confirm that this is not currently supported by Isaac sim

from omni.replicator.core.scripts.create.render_product_tiled

Note: Only a single tiled resolution is allowed per session. Creating more render products with different tiled resolution will cause the previous tiled render products to render incorrectly.
